### DISCUSSION TITLE
proposal: How to grow envd community

### DISCUSSION THREAD

**Discussion by gaocegege** [tensorchord/envd#1167] [General]
envd is growing, people are engaged, and we're committed to keeping this thing going. Thus we need to discuss how to form our community and make envd community healthy.

Some respects:

## community meeting

- Monthly community meeting (to sync with the community)
  - Maybe we need a meeting for the EU/Asia time zone and one for the US.
  - Ref [Flyte community meeting](https://www.youtube.com/watch?v=LoDDKpT92tM)
- Online bi-weekly office hours to make it possible to communicate with maintainers and contributors directly
- Quarterly roadmap planning meeting to discuss the roadmap of envd.

## meetups and talks

- Monthly TensorChord tea hour
- Quarterly MLOps meetup (optional)
  - We may not have the capacity for it. Let's see if we can collaborate with other MLOps communities.

## governance and leadership

- Contributors. A “contributor” could be anyone who comments on an issue or pull request, people who add value to the project (whether it’s triaging issues, writing code, or organizing events), or anybody with a merged pull request (perhaps the narrowest definition of a contributor).
- Reviewers. https://github.com/kubernetes/community/blob/master/community-membership.md#reviewer
- Approvers. https://github.com/kubernetes/community/blob/master/community-membership.md#approver

Some references:

- https://github.blog/2016-06-23-the-shape-of-open-source/
- https://opensource.guide/leadership-and-governance/

**Comment by lalawuu** [comment]
Here is a idea about Grow: https://github.com/tensorchord/envd/discussions/1182
